mangaki/mangaki/utils/knn.py

```diff
@@ -94,7 +97,12 @@ def get_common_traits(self, my_username, username):
             if abs(my) >= 1 and abs(their) >= 1:
                 print('%d.' % rank, works[work_id].title, my, their, '=', my * their)
 
-    def fit(self, X=[], y=[], all_dataset=False):
+    def fit(self, X=None, y=None, all_dataset=False):
+        if X is None:
+            X = []
+        if y is None:
+            y = []
```

Comment:
Ask @jilljenn — my understanding was: `X` represents a matrix (in maths, we often uppercase them, so it was "domain specific code style" that I tried to kept.)